packages/workflow/index.js

```diff
@@ -79,14 +99,7 @@ yargs
     build();
   })
 
-  .command(
```

Comment:
This had to be moved up outside of this `yargs` chain so that the `-v` and `--version` could be used. Else it would run the version stuff defined below which just prints the package version.